tailwind.config.js

```diff
@@ -48,10 +49,31 @@ module.exports = {
         "whitney": ["Whitney Medium", "sans-serif"],
         "whitney-bold": ["Whitney Semibold Regular", "sans-serif"],
       },
+      scrollPad: {
+        2: "2rem",
+        4: "4rem"
+      }
     },
   },
   variants: {
     extend: {},
   },
-  plugins: [require("@tailwindcss/typography"), require("@tailwindcss/forms")],
+  plugins: [
+    require("@tailwindcss/typography"), 
+    require("@tailwindcss/forms"),
+    plugin(function({ addUtilities, theme, e }) {
```

Comment:
Is it possible to simplify this by using a style like this? https://github.com/tailwindlabs/tailwindcss/blob/master/src/plugins/padding.js

Reply:
Discovered an easier way to do this I didn't know about - sorry Harry!

Reply:
Np, was fun to try learn about it